### ISSUE TITLE
Consecutive capital letters in jsx-pascal-case

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#328]
I like the new pascal case rule, but I've run into a couple of cases where I have consecutive capital letters and am wondering if perhaps this should be allowed. For instance:

```jsx
<CSSTransitionGroup>
```

**Comment by yannickcr** [comment]
I think this should also be allowed by the rule.

**Comment by lencioni** [comment]
Great! I'll submit a PR shortly.
